backend/bom-bom-server/src/main/java/me/bombom/api/v1/auth/handler/OAuth2LoginSuccessHandler.java

```diff
@@ -25,10 +29,18 @@ public void onAuthenticationSuccess(
         CustomOAuth2User oAuth2User = (CustomOAuth2User) authentication.getPrincipal();
         Member member = oAuth2User.getMember();
 
+        String redirectUrl = frontendBaseUrl;
+
+        HttpSession session = request.getSession(false);
+        String env = session != null ? (String) session.getAttribute("env") : null;
+        if ("local".equals(env)) {
+            redirectUrl = frontendLocalUrl;
+        }
+
         if (member == null) {
-            response.sendRedirect(frontendBaseUrl + "/signup");
+            response.sendRedirect(redirectUrl + "/signup");
         } else {
-            response.sendRedirect(frontendBaseUrl + "/");
+            response.sendRedirect(redirectUrl + "/");
         }
```

Comment:
p3

```suggestion
        if (member == null) {
            redirectUrl += "/signup";
        } else {
            redirectUrl += "/";
        }

        response.sendRedirect(redirectUrl);
```

이런 식으로 하고, 조건문 부분은 메서드로 분리해도 되지 않을까용?